src/Sql/Sql/help/Restore-AzSqlInstanceDatabase.md

```diff
@@ -56,26 +56,57 @@ Restore-AzSqlInstanceDatabase [-FromPointInTimeBackup] [-ResourceId] <String> -P
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### GeoRestoreFromGeoBackupSetNameFromResourceIdParameter
+```
+Restore-AzSqlInstanceDatabase [-FromGeoBackup] [-ResourceId] <String> -TargetInstanceDatabaseName <String>
+ -TargetInstanceName <String> -TargetResourceGroupName <String> [-AsJob]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### GeoRestoreFromGeoBackupSetNameFromNameAndResourceGroupParameter
+```
+Restore-AzSqlInstanceDatabase [-FromGeoBackup] [-Name] <String> [-InstanceName] <String>
+ [-ResourceGroupName] <String> -TargetInstanceDatabaseName <String> -TargetInstanceName <String>
+ -TargetResourceGroupName <String> [-AsJob] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
 ## DESCRIPTION
-The **Restore-AzSqlInstanceDatabase** cmdlet restores an instance database from a point in time in a live database.
+The **Restore-AzSqlInstanceDatabase** cmdlet restores an instance database from a geo-redundant backup or a point in time in a live database.
 The restored database is created as a new instance database.
 
 ## EXAMPLES
 
-### Example 1: Restore a instance database from a point in time
+### Example 1: Restore an instance database from a point in time
 ```
 PS C:\> Restore-AzSqlinstanceDatabase -Name "Database01" -InstanceName "managedInstance1" -ResourceGroupName "ResourceGroup01" -PointInTime UTCDateTime -TargetInstanceDatabaseName "Database01_restored"
 ```
 
 The command restores the instance database Database01 from the specified point-in-time backup to the instance database named Database01_restored.
 
-### Example 2: Restore a instance database from a point in time to another instance on different resource group
+### Example 2: Restore an instance database from a point in time to another instance on different resource group
 ```
 PS C:\> Restore-AzSqlInstanceDatabase -Name "Database01" -InstanceName "managedInstance1" -ResourceGroupName "ResourceGroup01" -PointInTime UTCDateTime -TargetInstanceDatabaseName "Database01_restored" -TargetInstanceName "managedInstance1" -TargetResourceGroupName "ResourceGroup02"
 ```
 
 The command restores the instance database Database01 on instance managedInstance1 on resource group ResourceGroup01 from the specified point-in-time backup to the instance database named Database01_restored on instance managedInstance2 on resource group ResourceGroup02.
 
+### Example 3: Geo-Restore an instance database
+```
+PS C:\>$GeoBackup = Get-AzSqlInstanceDatabaseGeoBackup -ResourceGroupName "ResourceGroup01" -InstanceName "managedInstance1" -Name "Database01"
+PS C:\> Restore-AzSqlDatabase -FromGeoBackup -GeoBackupObject $GeoBackup -TargetInstanceDatabaseName "Database01_restored" -TargetInstanceName "managedInstance1" -TargetResourceGroupName "ResourceGroup02"
+```
+
```

Comment:
I would modify the second line a little to demonstrate piping scenario: 
```PowerShell
PS C:\> $GeoBackup | Restore-AzSqlDatabase -FromGeoBackup -TargetInstanceDatabaseName "Database01_restored" -TargetInstanceName "managedInstance1" -TargetResourceGroupName "ResourceGroup02"
"managedInstance1"
```

Reply:
okay

Reply:
I see no changes.